### ISSUE TITLE
LOCoverfilesovertime : "Language" series has no matching column

### ISSUE THREAD

**Issue by mhow2** [crossminer/scava#223]
If I'm not wrong, either their should be a col with name `Language` or maybe there is no matter of _Language_ at all in this metric.  Could you advise @tdegueul and relabel the issue as appropriate ?

```json
  {
    "id": "LOCoverfilesovertime",
    "name": "LOC over files over time",
    "description": "This metric shows the distribution of physical lines of code over files over time.",
    "type": "LineChart",
    "datatable": {
      "rows": "$value",
      "cols": [
        {
          "name": "Date",
          "field": "$__date"
        },
        {
          "name": "Gini",
          "field": "$value"
        }
      ]
    },
    "x": "Date",
    "y": "Gini",
    "series": "Language",
    "x-label": "Date",
    "y-label": "Gini coefficient",
    "timeSeries": true,
    "metricId": "trans.rascal.LOC.genericLOCoverFiles.historic"
  },
```

**Comment by mhow2** [comment]
Hi @tdegueul , do you have a clue ? Thanks.

**Comment by mhow2** [comment]
bump :)

**Comment by tdegueul** [comment]
Ineed, this metric is only related to LOC and is language-agnostic. I don't see why it would use languages in the series.

**Comment by tdegueul** [comment]
Fixed in 489f92e40bf39ed0e9b53e4fb1327793e5dd6878.
